README.rst

```diff
@@ -11,122 +11,43 @@ so, grafanalib is for you.
 
 grafanalib lets you generate Grafana dashboards from simple Python scripts.
 
-Writing dashboards
-==================
-
-The following will configure a dashboard with a single row, with one QPS graph
-broken down by status code and another latency graph showing median and 99th
-percentile latency:
-
-.. code-block:: python
```

Comment:
I'm a little concerned about removing any example from the top level README. I think it would be good to keep at least a simple example for the benefit of people evaluating options.

Reply:
I agree that it'd be nice, unfortunately the `literalinclude` bit won't work on Github - it'll work on readthedocs though: https://github.com/weaveworks/grafanalib/pull/220/files#diff-22ab0e1b24df9ee7410dc1959a86d339R21